app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>SHOW INFO OF ONE USER</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <div class='container'>
+      <div class='card'>
+        <img class='userpic' src= <%= @user.photo %> >
+        <div class='leftcard'>
+          <h2 class='username'><%= @user.name %></h2>
+          <p class='count'>Number of posts: <%= @user.posts_counter >= 0 ? @user.posts_counter : '0' %></p><br>
```

Comment:
- [Optional] Let's kindly make sure our codebase is DRY and avoid repeating ourselves :+1: